java/api/src/main/java/org/ray/api/options/CallOptions.java

```diff
@@ -18,23 +18,14 @@ private CallOptions(Map<String, Double> resources, boolean useDirectCall) {
   public static class Builder {
 
     private Map<String, Double> resources = new HashMap<>();
-    private boolean useDirectCall = DEFAULT_USE_DIRECT_CALL;
 
     public Builder setResources(Map<String, Double> resources) {
       this.resources = resources;
       return this;
     }
 
-    // Since direct call is not fully supported yet (see issue #5559),
```

Comment:
There's another copy of these comments in `ActorCreationOptions.java`. Please remove it too.